apps/docs/src/content/reference/extras/instanced-spritesheet.mdx

```diff
@@ -0,0 +1,230 @@
+---
+{
+  order: 3,
+  category: '@threlte/extras',
+  sourcePath: 'packages/extras/src/lib/components/InstancedSprite/Spritesheet.svelte',
+  name: <Spritesheet>,
+  type: 'component',
+  componentSignature:
+    {
+      props:
+        [
+					{
+            name: 'dataUrl',
+            type: 'string',
+            required: false,
+            description: 'Url of a spritesheet json with metadata.'
+          },
+          {
+            name: 'textureUrl',
+            type: 'string',
+            required: false,
+            description: 'Url of a spritesheet image. This will get overwritten if you declare spritesheet meta with help of <File/> and <Animation/> child components.'
+          },
+					{
+						name: 'aseprite',
+						type: 'boolean',
+						required: false,
+						description: 'Set to true if you are using an Aseprite .json file in dataUrl.'
+          },
+					{
+						name: 'filter',
+						type: 'nearest | linear',
+						default: 'nearest',
+						required: false,
+						description: 'The texture filtering applied to the spritesheet.'
+          }
+        ],
+      events:
+        [
+          {
+            name: 'load',
+            payload: 'void',
+            description: 'Fires when spritesheet metadata and texture have been fully created or loaded.'
+          }
+        ]
+    }
+}
+---
+
+<Tip type="experimental">Experimental warning. The api may change significantly, performance and compatiblity untested on various hardware/OS.</Tip>
+
+**Utility component to declare or load a spritesheet for `<InstancedSprite>` component.**
+
+## Spritesheet
+
+Spritesheet declaration, two modes:
+- `rowColumn` where `w` is number of columns and `h` number of rows
+- `frameSize` where `w` is width and `h` is height of a **single frame in the spritesheet**
+
+{/* I'll center it later, don't yell at me */}
+Preview of row and column numbering (orange, outside) and frame numbering (blue, inside)
+![Spritesheet row-column](/images/docs/extras/instanced-sprite/spritesheet-rowcol.png)
+
+Animations are declared as a range of frameIds, for example the `attack` animation of the spritesheet above would look like this:
+```svelte
+<Spritesheet
+  let:File
+  let:Animation
+>
+  <File
+    textureUrl="/textures/sprites/cacodaemon.png"
+    options={{
+      type: 'rowColumn',
+      w: 8,
+      h: 4
+    }}
+  >
+    <Animation
+      name="attack"
+      frameRange={[8, 14]}
+    />
+  </File>
+</Spritesheet>
+```
+
+**One spritesheet file with multiple animations**
+<details>
+	<summary>Preview sprite</summary>
+	*cacodaemon.png*
+	![Multi image spritesheet](/textures/sprites/cacodaemon.png)
+</details>
+
+```svelte
+<Spritesheet
+  let:File
```

Comment:
If possible I'd avoid using component `let` syntax, since this will eventually be deprecated and removed from Svelte